rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1288,19 +1288,31 @@ def _log_metric_info(self) -> None:
 
     def _prepare_layers(self) -> None:
         self.text_name = TEXT
-        self._prepare_sequence_layers(self.text_name)
+        self._tf_layers[
+            f"{self.text_name}_sequence_layer"
+        ] = rasa_layers.RasaSequenceLayer(
+            self.text_name, self.data_signature[self.text_name], self.config
+        )
         if self.config[MASKED_LM]:
             self._prepare_mask_lm_layers(self.text_name)
```

Comment:
does it prepare mask loss?

Reply:
Yeah. I know we talked about moving some of this into my layers (but then we ran out of time and never discussed it properly). Currently, `RasaSequenceLayer` only returns sequence ids with input mask applied, and the rest happens outside of that layer, in DIET. How do you think this should work?

Reply:
that's good, could you please rename the function to represent that it creates loss layer